generators/client/templates/angular/src/main/webapp/app/core/util/alert.service.ts.ejs

```diff
@@ -81,7 +82,11 @@ export class AlertService {
 
     <%_ if (enableTranslation) { _%>
     if (alert.translationKey) {
-      alert.message = this.translateService.instant(alert.translationKey, alert.translationParams);
+      const translatedMessage = this.translateService.instant(alert.translationKey, alert.translationParams);
+      // if translation key exists
+      if (translatedMessage !== `${translationNotFoundMessage}[${alert.translationKey}]`) {
+        alert.message = translatedMessage;
+      }
```

Comment:
If no message use plain translationKey. Many times it is a full message.
```suggestion
      } else if (!alert.message) {
          alert.message = alert.translationKey;
      }
```

Can you add tests for alerts having translationKey at
https://github.com/jhipster/generator-jhipster/blob/22adabd1f9bf4c9562f723fb91045055d67123aa/generators/client/templates/angular/src/main/webapp/app/core/util/alert.service.spec.ts.ejs?

Reply:
Added the suggested fix and test cases for following scenarios:-
- If translationKey exists, should show translated message
- If translationKey doesn't exist, should show message if provided
- If message and translationKey both don't exist, show fallback to trasnlationKey